packages/n-m3u8dl-re/build.sh

```diff
@@ -0,0 +1,29 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/nilaoda/N_m3u8DL-RE"
+TERMUX_PKG_DESCRIPTION="Cross-Platform, beautiful and powerful stream downloader for DASH/HLS."
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="0.3.0-beta"
+TERMUX_PKG_SRCURL="https://github.com/nilaoda/N_m3u8DL-RE/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz"
+TERMUX_PKG_SHA256="6cbad6b6fda2733099cabb3cc1f89884984bdfb644416f14713d9bcdd0e55676"
+TERMUX_PKG_BUILD_DEPENDS="dotnet-runtime-9.0, dotnet-sdk-9.0, dotnet-targeting-pack-9.0"
```

Comment:
Consider making the version part a variable so it can be updated more easily in the future.
```suggestion
TERMUX_PKG_BUILD_DEPENDS="
dotnet-runtime-${TERMUX_DOTNET_VERSION},
dotnet-sdk-${TERMUX_DOTNET_VERSION},
dotnet-targeting-pack-${TERMUX_DOTNET_VERSION}
"
```

(This will obviously require moving up `TERMUX_DOTNET_VERSION=9.0`)